src/screens/MediaSeries/MediaSeries.tsx

```diff
@@ -0,0 +1,16 @@
+import type { MediaScreen } from '#types/screens';
+import { getSeriesId } from '#src/utils/media';
+import SeriesRouter from '#src/containers/SeriesRouter/SeriesRouter';
+
+/**
+ * This media screen is used to redirect a series linking media item to the first episode.
+ * Later, the watch history can also be used to determine the last watched episode to properly support continue
+ * watching with series.
+ */
+const MediaSeries: MediaScreen = ({ item }) => {
+  const seriesId = getSeriesId(item) || '';
+
+  return <SeriesRouter seriesId={seriesId} />;
```

Comment:
I am trying to figure out how to combine these 2 files, to prevent too much boilerplate.  It seems a shame to make this more complex just to support the legacy route.

Reply:
The MediaSeries screen redirects a linking media item to an episode (not a legacy route). However, we could theoretically skip this screen and obtain the series id in the `useGetSeriesId` hook. 

The only downside is that the URL will be the linking media item id instead of the current episode.

Reply:
I think having a screen for MediaSeries, even if it just redirects, does make sense, since, as you mention, it lines up with the linked media item.  The part I was trying to work around was how almost all of the logic on this screen is just in the child component, but I think renaming SeriesRouter to SeriesRedirect will help assuage that concern.